src/core/or/sendme.c

```diff
@@ -213,3 +213,40 @@ sendme_stream_data_received(edge_connection_t *conn)
   tor_assert(conn);
   return --conn->deliver_window;
 }
+
+/* Called when a relay DATA cell is packaged on the given circuit. If
+ * layer_hint is NULL, this means we are the Exit end point else we are the
+ * Client. Update the package window and return its new value. */
+int
+sendme_circuit_data_packaged(circuit_t *circ, crypt_path_t *layer_hint)
```

Comment:
I'd suggest adding "note" between sendme and stream/circuit in these functions' names, to make their purpose more clear.

Reply:
Done in commit ad100e855024251a